1exercise5.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <title>Task</title>
+    
+    </head>
+<body>
+
+    <script>
+function multiplyNumeric(menu) {
+    for (let key in menu) {
+    
+    if (typeof(menu[key]) === 'number') {
+        menu[key] = 2 * Number( menu[key] );
```

Comment:
`menu[key]` is a number. You've just checked it. Why do you need to convert it to `Number`?